server/controllers/student/user/get.js

```diff
@@ -0,0 +1,15 @@
+const getStudentInformation = require('../../../database/queries/getStudent');
+
+exports.get = (req, res) => {
+  getStudentInformation('12345')
+    .then((information) => {
+      res.send({
+        error: null,
+        data: information.rows,
+      });
+    })
+    .catch(err => res.send({
+      error: err,
```

Comment:
you can't send the error as you got it, you should handle it 
```suggestion
      error: ' Sorry we have some issues',
```